spec/models/allocation_spec.rb

```diff
@@ -152,7 +155,7 @@
         AllocationService.create_or_update(params)
 
         alloc = described_class.find_by(nomis_offender_id: nomis_offender_id)
-        alloc.deallocate_offender(movement_type)
+        AllocationService.offender_released(alloc)
```

Comment:
Is this still the right place for us to be testing this functionality?

Looks like we're now testing the behaviour of the `AllocationService.offender_released` method. Would it fit better into `spec/services/allocation_service_spec.rb`?